packages/fe-fpm-writer/templates/view/manifest.json

```diff
@@ -0,0 +1,34 @@
+{
+    "sap.ui5": {
+        <% if (locals.eventHandler) { %> 
+        "extends": {
+            "extensions": {
+                "sap.ui.controllerExtensions": {
+                    "sap.fe.templates.ListReport.ListReportController": {
+                        "controllerName": "<%- ns %>.<%- name %>"
+                    }
+                }
+            }
+        },
+        <% } %>
+        "routing": {
+            "targets": {
+                "<%- target %>": {
+                    "options": {
+                        "settings": {
+                            "views": {
+                                "paths": [
```

Comment:
You cannot assume that there is no view defined yet. If there would be any view definition already, you would destroy it.

Reply:
ok